mmpose/apis/inference.py

```diff
@@ -293,25 +312,28 @@ def _inference_single_pose_model(model,
 
 
 def inference_top_down_pose_model(model,
-                                  img_or_path,
```

Comment:
1. Please use `deprecated_api_warning` (https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/misc.py#L288) here to handle bc-breaking.
2. Please update all the places in mmpose where this API is used (demos, webcam API, tools, ...)

Reply:
OK, I will double-check it.